Cargo.toml

```diff
@@ -66,7 +66,7 @@ opt-level = 1
 typed-store = "0.1.0"
 typed-store-derive = "0.1.0"
 telemetry-subscribers = { version = "0.2.0", features = ["jaeger", "tokio-console"] }
-mysten-network = "0.1.0"
+mysten-network = { git = "https://github.com/MystenLabs/mysten-infra.git", rev = "8ad649d006a51f312bc415cb6668e3bcb602f050" }
```

Comment:
This should really be solved by publishing a new mysten-network (https://github.com/MystenLabs/mysten-infra/pull/173)